apps/rss/templates/article.jade

```diff
@@ -32,7 +28,9 @@ for section, i in article.get('sections')
           img(src=resize(artwork.image, { width: 500 } ))
           p= artwork.title + ", " + artwork.date + ". "
             br
-            if artwork.artist
+            if artwork.artists
+              = toSentence(_.pluck(artwork.artists, 'name'))
```

Comment:
yep so the result looks like: "Eve, Christina and Kana" http://gabceb.github.io/underscore.string.site/#toSentence